pkg/manager/member/utils.go

```diff
@@ -83,20 +83,7 @@ func SetStatefulSetLastAppliedConfigAnnotation(set *apps.StatefulSet) error {
 
 // SetLastAppliedConfigAnnotation set last applied config info to Statefulset's annotation and the podTemplate's annotation
 func SetLastAppliedConfigAnnotation(set *apps.StatefulSet) error {
```

Comment:
remove `SetLastAppliedConfigAnnotation` entirely?

Reply:
make sense, I bet there's no external usage of this method, though public

Reply:
removed in https://github.com/pingcap/tidb-operator/pull/1414/commits/e2868a6a7e5843f57cd763867cb995c4edf78b76